python/ray/serve/backend_worker.py

```diff
@@ -140,7 +141,7 @@ def wrap_to_ray_error(exception):
         return ray.exceptions.RayTaskError(str(e), traceback_str, e.__class__)
 
 
-def ensure_async(func):
+def ensure_async(func: Callable) -> Callable:
```

Comment:
-> AsyncCallable

Reply:
Couldn't find AsyncCallable in the project or on google, I made it Coroutine instead

Reply:
Oops. Sorry I confused types in `typing` and those in `inspect`. `Coroutine` is not actually correct. `Coroutine` are things you get after call call an `async` func and it can be awaited on. We are typing `async` in this case. Can we just revert the change, back to `Callable`?

Reply:
No problem, done